doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -50,51 +50,52 @@ Synopsis
 ========
 In general, the command to mount CephFS via kernel driver looks like this::
 
-    mount -t ceph {device-string}:{path-to-mounted} {mount-point} -o {key-value-args} {other-args}
+    mount -t ceph {device-string}={path-to-mounted} {mount-point} -o {key-value-args} {other-args}
 
 Mounting CephFS
 ===============
 On Ceph clusters, CephX is enabled by default. Use ``mount`` command to
 mount CephFS with the kernel driver::
 
     mkdir /mnt/mycephfs
-    mount -t ceph :/ /mnt/mycephfs -o name=foo
+    mount -t ceph <name>@<fsid>.<fs_name>=/ /mnt/mycephfs
 
-The key-value argument right after option ``-o`` is CephX credential;
-``name`` is the username of the CephX user we are using to mount CephFS. The
-default value for ``name`` is ``guest``.
+``name`` is the username of the CephX user we are using to mount CephFS. ``fsid`` is
+the FSID of the ceph cluster. ``fs_name`` is the file system to mount. Monitor hosts
+are (optionally) passed as an mount option parameter. If using mount helper, ``fsid``
+is optional as the mount helper can fill it in by reading the ceph configuration
+file::
+
+  mount -t ceph <name>@<fs_name>=/
```

Comment:
I feel there should be at least one example with everything filled in even if the mount helper does it. Also that the "fsid" comes from `ceph fsid` and not some cephfs command.

Reply:
Or refer (link) to the mount page for examples.